### PR TITLE
Minor dep update

### PR DESCRIPTION
**Pull request by MischaPanch**
Support gymnasium >=0.28

### CODE REVIEW

**Comment by MischaPanch** [comment]
@Trinkle23897 The support of older gymnasium should be uncontroversial, so it's mainly the readme which needs a look. I modified the top-level alert, added a second one, and also added appliedAI to the acknowledgments section. Maybe the statement about long-term support is attractive for some users :)

**Comment by MischaPanch** [comment]
Torch and poetry still don't like each other, even in torch 2.1.0 :( 

I added a section in the readme on how to install torch until they fix this

File: pyproject.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ packaging = "*"
 pettingzoo = "^1.22"
 tensorboard = "^2.5.0"
 # Torch 2.0.1 causes problems, see https://github.com/pytorch/pytorch/issues/100974
-torch = "^2.0.0, !=2.0.1"
+torch = "^2.0.0, !=2.0.1, !=2.1.0"
```
Comment (Trinkle23897) [inline pyproject.toml:39]: may i know why?

**Review by Trinkle23897 — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (MischaPanch) [reply pyproject.toml:39]: See my message above and the failing CI tests

Comment (MischaPanch) [reply pyproject.toml:39]: And the new link in the readme

**Comment by MischaPanch** [comment]
Woops, I clicked the button for auto-merge after checks pass, but it merged immediately for some reason. Sorry for that!

If CI does fail for some unexpected reason, I'll fix it immediately

**Comment by Trinkle23897** [comment]
np

**Pull request merged**